src/hooks/modules/useMultipartUploader.ts

```
@@ -595,6 +626,67 @@ export function useMultipartUploader(props: {
+          parent = props.rootPath || parentPath || '/'
+          if (parent.length > 1 && parent.endsWith('/')) {
+            parent = parent.slice(0, -1)
+          }
+          const pathParts = relativePath.split('/')
+          pathParts.pop()
+          const folderPath = pathParts.join('/')
+          if (folderPath) {
+            parent = `${parent}/${folderPath}`
+          }
+        } else {
+          parent = props.rootPath || parentPath || '/'
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)